safe-global/web-core
# feat: add "Messages" to transactions

## What it solves

Resolves #1155

## How this PR fixes it

- `/transactions/messages` route has been added with a placeholder
- `/transactions/msg?id={{messageHash}}` has been added
- "Messages" has been added to the transaction navigation tabs

## How to test it

- Open the transactions section of the Safe and observe the "Messages" tab. Clicking on it should navigate to `/transactions/messages` and display the placeholder.
- Open `/transactions/msg?id={{messageHash}}&safe={{shortName}}:{{address}}` and observe the given `messageHash` displayed.

## Screenshots

![image](https://user-images.githubusercontent.com/20442784/201907802-3491eaa0-b2c7-4bbf-b54c-f1b9133737a3.png)

![image](https://user-images.githubusercontent.com/20442784/201907835-69fbcbf7-a811-4088-a9a2-cc85db211f63.png)

![image](https://user-images.githubusercontent.com/20442784/201907932-5b68158b-c338-4c6a-96e4-98a3a3f03c57.png)
